realm/src/main/java/io/realm/Realm.java

```diff
@@ -755,6 +906,33 @@ private static void initializeRealm(Realm realm) {
     }
 
     /**
+     * Tries to update an existing object defined by its primary key with new JSON data. If no existing object could be
+     * found a new object will be saved in the Realm. This must happen inside a transaction.
```

Comment:
"inside" -> "within"?

According to http://en.wiktionary.org/wiki/within "within" means "before a specific duration ends" and the transaction is the "specific duration".